transport/src/main/java/io/netty/channel/embedded/EmbeddedChannelId.java

```diff
@@ -40,7 +40,7 @@ public String asLongText() {
     }
 
     @Override
-    public int compareTo(ChannelId o) {
+    public int compareTo(final ChannelId o) {
```

Comment:
Just because it not really adds anything to the patch IMHO... that said I don't care too much